tokio/src/net/addr.rs

```diff
@@ -33,13 +33,16 @@ impl sealed::ToSocketAddrsPriv for SocketAddr {
 
 // ===== impl str =====
 
+#[cfg(feature = "dns")]
```

Comment:
Always include `ToSocketAddr` but only implement for string types if the `dns` feature flag is enabled.